src/nodes/VolumeRendering/DepthMappedVolumeStyle.js

```diff
@@ -0,0 +1,395 @@
+                
+                this.uniformVec4fXSection._vf.name = 'uXSection';
+                this.uniformVec4fXSection._vf.type = 'SFVec4f';
+                this.uniformVec4fXSection._vf.value = this.xSection;
+                unis.push(this.uniformVec4fXSection);
+                
+                return unis;
+            },
+            
+            textures: function() {
+                var texs = [];
+                
+                var tex = this._cf.transferFunction.node;
+                if (tex) {
+                    tex._vf.repeatS = false;
+                    tex._vf.repeatT = false;
+                    texs.push(tex);
+                }
+                
+                tex = this._cf.depthTexture.node;
+                if(tex) {
+                    tex._vf.repeatS = false;
+                    tex._vf.repeatT = false;
+                    texs.push(tex);
+                }
+                
+                return texs;
+            },
+            
+            styleUniformsShaderText: function() {
+                var uniformsText = "uniform mat4 projectionMatrix;\n";
+                uniformsText += "uniform float uCubeSize;\n";
+                uniformsText += "uniform float uIsoSurfaceCutoffValue;\n";
+                uniformsText += "uniform float uTransparency;\n";
+                uniformsText += "uniform float uAccumFactor;\n";
+                uniformsText += "uniform float uBrightness;\n";
+                uniformsText += "uniform vec4 uXSection;\n";
+                uniformsText += "uniform vec2 uIntensityLimits;\n";
+                if (this._cf.transferFunction.node) {
+                        uniformsText += "uniform sampler2D uTransferFunction;\n";
+                }
+                if (this._cf.depthTexture.node) {
+                        uniformsText += "uniform sampler2D uDepthTexture;\n";
+                }
+                
+                return uniformsText;
+            },
+
+            fragmentShaderText: function(numberOfSlices, slicesOverX, slicesOverY){
+                var shader = 
+                this._parentNodes[0].fragmentPreamble+
+                this._parentNodes[0].defaultUniformsShaderText(numberOfSlices, slicesOverX, slicesOverY, true)+
+                this.styleUniformsShaderText()+
+                this._parentNodes[0].texture3DFunctionShaderText;
+                
+                var shaderLoop = "";
+                
+                shaderLoop +=
+                    "void main()\n"+
+                    "{\n"+
+                    "  vec3 cam_pos = vec3(modelViewMatrixInverse[3][0], modelViewMatrixInverse[3][1], modelViewMatrixInverse[3][2]);\n"+
+                    "  cam_pos = cam_pos/dimensions+0.5;\n"+
+                    
+                    "  const float steps = 200.0;\n"+ // take care! ANGLE does loop unrolling, setting this too high might not give any compile error but could give weird results
```

Comment:
Could this be "60.0" steps, in order to be consistent with the other volume rendering nodes?